docs/change-log/index.md

```diff
@@ -10,7 +10,23 @@
 - [v1.0.6](#v1-0-6-16-august-2022) | [v1.0.5](#v1-0-5-11-august-2022) | [v1.0.4](#v1-0-4-13-june-2022) | [v1.0.3](#v1-0-3-7-june-2022) | [v1.0.2](#v1-0-2-19-may-2022) | [v1.0.0](#v1-0-0-22-april-2022)
 - [Open issues](#open-issues)
 - [Limitations](#limitations)
-- [Deprecations](#deprecations-and-removed-code)
+- [Deprecations and removed code](#deprecations-and-removed-code)
+ 
+
+
+## v1.6.2 (29 March 2024)
+
+###  Closed issue
+| ID          |Description                                                               |
+|----------|---------------------------------------------------------------------------|
+|ML-5808|Changing the project owner to a user that shares a name with another user is successful.|
+|ML-5982|A user with the 'project security admin' policy can now add her/himself to a project.|
```

Comment:
can be omitted as it is fixed by ML-5907

Reply:
@liranbg remove both 5808 and 5982?

Reply:
remove only 5982